themes/godotengine/pages/fbx-import.htm

```diff
@@ -0,0 +1,106 @@
+title = "FBX Import"
+url = "/fbx-import"
+layout = "more"
```

Comment:
```suggestion
layout = "more-standalone"
```

You can use `more-standalone` now to have a similar page but without the sub-navigation tabs.